Cargo.toml

```diff
@@ -49,6 +49,8 @@ serde_json = "<1.0.45"
 serde_test = "1"
 secp256k1 = { version = "0.24.0", features = [ "recovery", "rand-std" ] }
 bincode = "1.3.1"
+# avoid backtrace crate because of MSRV
+io_check = { version = "0.1.2", default-features = false }
```

Comment:
Why don't we feature gate the tests an only run them with a stable toolchain, that way we can have backtrace. We don't loose anything by not testing read/write with MSRV, right?

Reply:
Yeah, I wasn't sure (and still am not) how to do it, this PR was meant mainly as demonstration.